publisher/socials/twitter.py

```
@@ -0,0 +1,121 @@
+import tweepy
+
+class TwitterAccount(SocialNetworkBase):
+    full_name = 'Twitter'
+    
+    consumer_key = encrypt(models.CharField(max_length=255))
+    consumer_secret = encrypt(models.CharField(max_length=255))
+    access_token = encrypt(models.CharField(max_length=255))
+    access_token_secret = encrypt(models.CharField(max_length=255))
+    
+    cckeditor_config = copy.deepcopy(SocialNetworkBase.ckeditor_config)
+    ckeditor_config['toolbar'].append([
```

Ruff (0.12.2)

Undefined name `ckeditor_config`

(F821)